test/live/live_test.go

```diff
@@ -52,17 +53,64 @@ func TestLiveErrorHandlingAttestFnCall(t *testing.T) {
 
 	test.NewTestscriptTest(t, projectDir).
 		ExecuteMakeTarget("build").
-		CopyFile("tmp/x.wasm").
+		CopyProjectFile("tmp/x.wasm").
 		RenderTemplateStringFromEnvWithCleanup(
 			liveTestConfigTemplate,
 			"config.toml",
 			requiredEnvVars).
-		CopyFile("successFunc.json").
-		CopyFile("errorFunc.json").
-		CopyFile("panicFunc.json").
+		CopyProjectFile("successFunc.json").
+		CopyProjectFile("errorFunc.json").
+		CopyProjectFile("panicFunc.json").
 		Run(filepath.Join(scriptDir, projectName+".txtar"))
 }
 
+func TestLiveErrorHandlingCombined(t *testing.T) {
```

Comment:
Forgot to add live versions of the combo tests the first time around. Notice that I left out TWAP because it's templates are actually hardcoded to only work with the local server. There is a card for adding twap live tests so that will be handled in a separate PR